src/styles/extends.scss

```diff
@@ -0,0 +1,21 @@
+%cell-formatting {
+  display: flex;
+  box-sizing: border-box;
+  background-color: #eee;
+  border: 1px solid black;
+
+  justify-content: center;
+  text-align: center;
+  justify-items: center;
+  align-items: center;
+}
+
+%calendar-formatting {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  justify-items: center;
+  margin: 100px auto;
```

Comment:
in my opinion - task 'Display a calendar in the middle of the screen' is matters in the middle of height and in the middle of width. you did margin top and bottom 100px. it's not the same